String/anagram/anagram.cpp

```diff
@@ -0,0 +1,80 @@
+#include <iostream>
+#include <string.h>
+using namespace std;
+ 
+void quickSort(char *arr, int si, int ei);
+ 
+bool areAnagram(char *str1, char *str2)
+{
+   
+    int n1 = strlen(str1);
+    int n2 = strlen(str2);
+ 
+    if (n1 != n2)
+      return false;
+ 
+   
+    quickSort(str1, 0, n1 - 1);
+    quickSort(str2, 0, n2 - 1);
+ 
+    
+    for (int i = 0; i < n1;  i++)
+       if (str1[i] != str2[i])
+         return false;
+ 
+    return true;
+}
+void exchange(char *a, char *b)
+{
+    char temp;
+    temp = *a;
+    *a   = *b;
+    *b   = temp;
+}
+ 
+int partition(char A[], int si, int ei)
+{
+    char x = A[ei];
+    int i = (si - 1);
+    int j;
+ 
+    for (j = si; j <= ei - 1; j++)
+    {
+        if(A[j] <= x)
```

Comment:
Not sorting?

Reply:
Actually the statement is :Program to check whether two strings are anagram or not  , using sorting.
First sort the strings and then compare them.